server/controllers/index.js

```diff
@@ -7,12 +7,45 @@ const { emitRooms, disconnectSocket, closeRoom, disconnectAllSockets } = require
 let currentUsers = [];
 let db;
 
+const oauthClientId = process.env.CLIENT_ID;
+const oauthSecret = process.env.CLIENT_SECRET;
+const redirectURL = process.env.NODE_ENV === 'development' ? "http://localhost:3000/lobby" : "/lobby";
```

Comment:
I still think we shouldn't have to do this but I'm not sure what else to suggest. The only other thing I can think of is to remove the `proxy` from the `package.json` and bring back `setupProxy.js` with a new proxy for the oauth end point. I'm not convinced it's going to work but you can give it a go if you like.

Reply:
I'll give it a go

Reply:
It didn't work